packages/config-loader/src/config-loader.ts

```diff
@@ -0,0 +1,12 @@
+declare const __webpack_public_path__: string; 
+
+export async function loadConfig(
+    fileName: string, 
+    envName: string, 
+) {
+    const configFileName = envName ? `${fileName}.${envName}` : fileName;
+    const url = `${__webpack_public_path__}configs/${configFileName}.json`;
```

Comment:
the `configs` key should be passed to this method, since it's dynamic at build time

Reply:
can we discuss it?